frontend/src/components/pages/RegistrantExperience/RegistrationSteps/QuestionGroups/MultipleChoiceGroup.tsx

```diff
@@ -10,28 +10,46 @@ import {
   VStack,
 } from "@chakra-ui/react";
 import { FormQuestion } from "../../../../../types/CampsTypes";
+import RequiredAsterisk from "../../../../common/RequiredAsterisk";
 
 type MultipleChoiceGroupProps = {
   formResponses: Map<string, string> | undefined;
   question: FormQuestion;
-  handleMultipleChoiceUpdate: (choice: string, question: FormQuestion) => void;
+  handleMultipleChoiceChange: (choice: string, question: FormQuestion) => void;
   nextClicked: boolean;
+  editing?: boolean;
 };
 
 const MultipleChoiceGroup = ({
   formResponses,
   question,
-  handleMultipleChoiceUpdate,
+  handleMultipleChoiceChange,
   nextClicked,
+  editing = false,
 }: MultipleChoiceGroupProps): React.ReactElement => {
-  const invalid =
-    nextClicked && !formResponses?.get(question.question) && question.required;
+  let invalid = false;
+  if (!editing)
+    invalid =
+      nextClicked &&
+      !formResponses?.get(question.question) &&
+      question.required;
+  else invalid = !formResponses?.get(question.question) && question.required;
 
   return (
-    <FormControl isRequired={question.required} isInvalid={invalid}>
+    <FormControl isInvalid={invalid}>
       <FormLabel>
         <Text textStyle={{ sm: "xSmallBold", lg: "buttonSemiBold" }}>
-          {question.question}
+          {question.question}{" "}
+          {question.required && (
+            <Text
+              as="span"
+              color="text.critical.100"
+              fontSize="xs"
+              verticalAlign="super"
+            >
+              <RequiredAsterisk />
+            </Text>
+          )}
```

Comment:
same comment as before - how come isRequired doesn't cut it?

Reply:
See above.